packages/terra-list/src/ListSubsectionHeader.module.scss

```
@@ -0,0 +1,79 @@
+@import '~terra-mixins';
+
+// Subsection Header
+// ========================================================
+:local {
+  .subsection-header {
+    background-color: var(--terra-list-v3-subsection-header-background-color, #dedfe0);
+    border-bottom: var(--terra-list-v3-subsection-header-border-bottom, 1px solid #c8cacb);
+    padding-bottom: var(--terra-list-v3-subsection-header-padding-bottom, 0.3571428571rem);
+    padding-left: var(--terra-list-v3-subsection-header-padding-left, 1.2142857143rem);
+    padding-top: var(--terra-list-v3-subsection-header-padding-top, 0.3571428571rem);
+    width: 100%;
+  }
+
+  .title {
+    color: var(--terra-list-v3-subsection-header-title-color, #1c1f21);
+    font-size: var(--terra-list-v3-subsection-header-title-font-size, 0.8571428571rem);
+    font-weight: var(--terra-list-v3-subsection-header-title-font-weight, bold);
+    line-height: var(--terra-list-v3-subsection-header-title-line-height, 1.1666666667);
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    word-break: break-all;
```

Let's change this to the following:

```suggestion
    overflow-wrap: break-word; /* Modern browsersrs */
    word-wrap: break-word; /* For IE 10 and IE 11 */
```

---

Updated.